packages/editor-ui/src/components/WorkflowActivator.vue

```diff
@@ -61,13 +61,11 @@ import { getActivatableTriggerNodes } from '@/utils/nodeTypesUtils';
 
 export default defineComponent({
 	name: 'WorkflowActivator',
-	mixins: [workflowActivate],
 	props: ['workflowActive', 'workflowId'],
-	setup(props, ctx) {
+	setup() {
 		return {
 			...useToast(),
-			// eslint-disable-next-line @typescript-eslint/no-misused-promises
-			...workflowActivate.setup?.(props, ctx),
+			...useWorkflowActivate(),
```

Comment:
Nitpick: Having named imports and exports instead of spread would make it more clear what's actually used from the composable:
```ts
const { updateWorkflowActivation, updatingWorkflowActivation } = useWorkflowActivate();
return {
	...useToast(),
	updateWorkflowActivation,
	updatingWorkflowActivation,
};
```